uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/sickness-leave-request-instance_test.js

```diff
@@ -114,5 +114,41 @@ define([
         expect(instance.required_documents).toEqual('');
       });
     });
+
+    describe('with required documents', function() {
+      beforeEach(function() {
+        instance = instance.init({}, false);
+      });
+
+      describe('for new leave request instance', function() {
+        it('is initialized to empty string', function() {
+          expect(instance.required_documents).toEqual('');
+        });
+      });
+
+      describe('when added by user', function() {
+        beforeEach(function () {
+          instance.toggleDocument("1");
+          instance.toggleDocument("2");
+        });
+
+        it('leave request instance has required documents', function() {
+          expect(instance.required_documents).toEqual('1,2');
+        });
+      });
+
+      describe('when removed by user', function() {
```

Comment:
Same thing, you are not in a controller anymore, so there shouldn't be any reference to a user doing anything